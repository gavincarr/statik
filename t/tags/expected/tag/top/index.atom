<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <title>My Bl√∏g</title> 
  <subtitle>Mutterings from the &lt;dungeon&gt;</subtitle> 
  <link href="http://example.com/testblog/tag/top/"/>
  <updated>2011-01-04T14:15:16+0000</updated>
  <author>
    <name>Jane Author</name>
  </author> 
  <id>tag:example.com,2010:/testblog/tag/top/</id>
  <generator uri="https://github.com/gavincarr/statik" version="0.1">Statik</generator>
  <entry>
    <title>Post the fourth</title>
    <link href="http://example.com/testblog/fourth"/>
    <id>tag:example.com,2011-01-04:/testblog/fourth</id>
    <updated>2011-01-04T14:15:16+0000</updated>
  </entry>
  <entry>
    <title>Post the third</title>
    <link href="http://example.com/testblog/third"/>
    <id>tag:example.com,2011-01-03:/testblog/third</id>
    <updated>2011-01-03T20:20:20+0000</updated>
  </entry>
  <entry>
    <title>Post the first</title>
    <link href="http://example.com/testblog/first"/>
    <id>tag:example.com,2011-01-01:/testblog/first</id>
    <updated>2011-01-01T12:00:00+0000</updated>
  </entry>
</feed>
