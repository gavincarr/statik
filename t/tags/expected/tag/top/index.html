<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>My Bløg</title>
<link rel="stylesheet" type="text/css" href="style-original.css" />
<link rel="alternate" type="application/atom+xml" title="My Bløg feed" href="http://example.com/testblog/index.atom" />
</head>
<body>
<div id="header">
<h1>My Bløg</h1>
<p class="tagline">Mutterings from the &lt;dungeon&gt;</p>
</div>
<div id="body">
<div id="article">
<h2><a href="http://example.com/testblog/fourth" rel="bookmark" title="link to this post">Post the fourth</a></h2>
<div id="timestamp">2011-01-04</div>
<p>Morbi volutpat leo in ligula. Integer vel magna. Quisque ut magna et
nisi bibendum sagittis. Fusce elit ligula, sodales sit amet, tincidunt
in, ullamcorper condimentum, lectus. Aliquam ut massa. Suspendisse
dolor. Cras quam augue, consectetuer id, auctor ut, tincidunt a, velit.
Quisque euismod tortor sed nulla. Nunc dapibus, nisi et iaculis feugiat,
leo ipsum venenatis enim, a nonummy magna ante vitae diam. Proin sapien.
Duis eleifend. Praesent tempor velit molestie neque. Lorem ipsum dolor
sit amet, consectetuer adipiscing elit. Sed mollis justo eget augue.
Donec tempus, urna a congue ultrices, lacus magna convallis nulla, non
ultrices metus justo et purus. In leo lorem, dapibus at, volutpat sed,
posuere a, justo. Donec varius, erat in placerat pharetra, lorem est
gravida erat, nec accumsan turpis erat sed velit. Duis malesuada, lacus
sit amet dictum lobortis, arcu velit sodales lectus, ac mattis ipsum
lacus in magna. Curabitur sed ante ac enim consequat porttitor.
Suspendisse bibendum turpis a magna.</p>

<p>Fusce interdum. Maecenas eu elit sed nulla dignissim interdum. Sed
laoreet. Aenean pede. Phasellus porta. Ut dictum nonummy diam. Sed a
leo. Cras ullamcorper nibh. Sed laoreet. Praesent vehicula suscipit
ligula. Morbi ullamcorper.</p>
</div>
<div id="article">
<h2><a href="http://example.com/testblog/third" rel="bookmark" title="link to this post">Post the third</a></h2>
<div id="timestamp">2011-01-03</div>
<p>At vero eos et accusamus et iusto odio dignissimos ducimus qui
blanditiis praesentium voluptatum deleniti atque corrupti quos dolores
et quas molestias excepturi sint occaecati cupiditate non provident,
similique sunt in culpa qui officia deserunt mollitia animi, id est
laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita
distinctio. Nam libero tempore, cum soluta nobis est eligendi optio
cumque nihil impedit quo minus id quod maxime placeat facere possimus,
omnis voluptas assumenda est, omnis dolor repellendus.</p>

<p>Temporibus autem
quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet
ut et voluptates repudiandae sint et molestiae non recusandae. Itaque
earum rerum hic tenetur a sapiente delectus, ut aut reiciendis
voluptatibus maiores alias consequatur aut perferendis doloribus
asperiores repellat.</p>
</div>
<div id="article">
<h2><a href="http://example.com/testblog/first" rel="bookmark" title="link to this post">Post the first</a></h2>
<div id="timestamp">2011-01-01</div>
<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
occaecat cupidatat non proident, sunt in culpa qui officia deserunt
mollit anim id est laborum.</p>
</div>
</div>
<div id="footer">
Copyright 2011 Jane Author.
</div>
</body>
</html>
